src/Entity/Activity/Registration.php

```diff
@@ -100,6 +101,13 @@ class Registration
      */
     private $comment;
 
+    /**
+     * @ORM\Embedded(class="ExternalRegistrant")
+     *
+     * @var ?ExternalRegistrant
+     */
+    private $external;
```

Comment:
That's definitely an option, however I feel it's a bit verbose. Can we come up with anything shorter?

Reply:
I updated this to $externalPerson now, which is more clear but also indicates its relation to $person. In the future, it would be best to update $person to $internalPerson (or both to $internal/$externalContact or something), but again, this would affect so much code that such a change is unfeasible here.